### PR TITLE
Refer to valid `sampler type` values (doc link)

### PR DESCRIPTION
**Pull request by sjoerdtalsma**
## Which problem is this PR solving?
- Make it easier to find the valid sampler types from the java client README

## Short description of the changes
- Link to jaeger documentation on client samling configuration.

### CODE REVIEW

File: jaeger-core/README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ JAEGER_PROPAGATION | no | Comma separated list of formats to use for propagating
 JAEGER_REPORTER_LOG_SPANS | no | Whether the reporter should also log the spans
 JAEGER_REPORTER_MAX_QUEUE_SIZE | no | The reporter's maximum queue size
 JAEGER_REPORTER_FLUSH_INTERVAL | no | The reporter's flush interval (ms)
-JAEGER_SAMPLER_TYPE | no | The sampler type
+JAEGER_SAMPLER_TYPE | no | [The sampler type](https://www.jaegertracing.io/docs/latest/sampling/#client-sampling-configuration)
```
[the remark this answers is not in the document]
Comment (sjoerdtalsma) [reply jaeger-core/README.md]: You're right. Updated the link

**Comment by sjoerdtalsma** [comment]
@objectiser 
>LGTM - can you sign the comment though. Thanks.

Sure, want me to just add a "Signed off by" line to the main comment of this PR or must I ammend my commits? (I made the change through the github web UI, so not that comfortable here :wink: )

**Comment by objectiser** [comment]
@sjoerdtalsma You will need to amend the commit, you could squash and rebase at the same time.

**Comment by sjoerdtalsma** [comment]
-> Any idea how to do this through github's UI?

The commits are signed by github because I use 2PA I think (see `Verified` with the commits).
Unfortunately I can't sign locally, as I don't have my PGP keys with me on the computer I'm currently using.

**Comment by sjoerdtalsma** [comment]
I could re-do the change this evening on my laptop though, so don't spend too much time on this :wink:

**Comment by sjoerdtalsma** [comment]
Rebased, squashed, amended, signed, signed off and force-pushed...  :smile:

**Pull request merged**